.github/workflows/inspektor-gadget.yml

```diff
@@ -504,6 +498,14 @@ jobs:
         outputs: type=registry,name=${{ steps.set-repo-determine-image-tag.outputs.container-repo }},push=true,push-by-digest=true
         cache-from: type=local,src=/tmp/.buildx-cache-new
         platforms: ${{ matrix.os }}/${{ matrix.platform }}
+    - name: Setup oras
+      if: github.event_name != 'pull_request'
+      uses: oras-project/setup-oras@v1
+    - name: Attach SBOM to container image
+      if: github.event_name != 'pull_request'
+      id: gadget-container-images-attach-sbom
+      run: |
+        oras attach ${{ steps.set-repo-determine-image-tag.outputs.container-repo }}@${{ steps.publish-gadget-container-images.outputs.digest }} /tmp/gadget-container-image-${{ matrix.os }}-${{ matrix.platform }}/sbom_cyclonedx.json --disable-path-validation --artifact-type example/sbom
```

Comment:
Could you add a comment referencing some documentation that explains the best practices for attaching the SBOM as artifact?

In particular, I wonder about the "example/sbom" artifact type.

What is `--disable-path-validation`?